pkg/kubelet/container/ref_test.go

```diff
@@ -22,6 +22,7 @@ import (
 	"k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/kubernetes/pkg/api"
+	_ "k8s.io/kubernetes/pkg/api/install"
```

Comment:
Can you add a TODO here saying we can remove this import if we change all `api.Registry.GroupOrDie(v1.GroupName).GroupVersion.String(),` to `"v1"`?